### ISSUE TITLE
What constitutional changes would you advocate for and why?

### ISSUE THREAD

**Issue by nyabinary** [NixOS/SC-election-2024#35]
### Question

If you had the opportunity to amend the constitution, what specific changes would you propose, and why do you believe these changes are necessary?

### Reminder of the Q&A rules

Please adhere to the [Q&A guidelines and rules](https://github.com/NixOS/SC-election-2024/blob/main/doc/qna.md)

**Comment by cafkafk** [comment]
I'd prohibit community members from serving on the board and steering committee at the same time, as I think it concentrates too much power on that individual, and counteracts the checks and balances built into the constitution.

**Comment by Gabriella439** [comment]
I would change the voting process for the Steering Committee to use [approval voting](https://electionscience.org/library/approval-voting/) instead of using [CIVS](https://civs1.civs.us/proportional.html).  Specifically, given N open slots on the steering committee in any given election, voters would be allowed to vote for as many candidates as they liked (maximum 1 vote per eligible candidate) and the top N vote getters would be appointed.  Simple!

The main reasons I would propose this are:

- Approval voting is (much) easier to reason about

  Approval voting's simplicity makes it not only easier to implement, but (more importantly) it is also easier for voters to reason about.  In particular, the easier it is to explain a voting methodology then the easier it is for voters to reason about both past outcomes (increasing their trust that the best candidates were selected) and future outcomes (understanding how their potential vote would affect the outcome).

- [Approval voting yields Condorcet winners in practice](https://rangevoting.org/AppCW.html)

  In fact, the above article mentions that Approval voting is perhaps *more* likely to elect Condorcet winners than Condorcet voting methods.

  > Indeed – counterintuitively – it might actually be that Approval Voting is more likely to elect the Condorcet Winner in practice, than Condorcet methods! (Indeed, [experiments](https://rangevoting.org/StratHonMix.html) indicate that happens.)
  >
  > Why? Because in approval voting it is quite rare that strategically voting dishonestly, is wise. (And when it is wise, it is even [rarer](https://rangevoting.org/RVstrat2.html) that people will actually realize it and do it.) In other words, with Approval, people will tend to honestly order the candidates, and the only strategic decision they'll make is where to locate their approval "threshold."

- Approval voting satisfies the monotonicity criterion

  A voting methodology satisfies the [monotonicity criterion](https://en.wikipedia.org/wiki/Monotonicity_criterion) if voting for a candidate always increases their chances of winning.  Approval voting satisfies the monotonicity criterion and [Condorcet voting does not](https://www.sciencedirect.com/science/article/abs/pii/S0165176515000889)!  Yes, you read that right: voting for or ranking a candidate higher in Condorcet voting can sometimes harm their chances of winning!  This is what happens when you use an overengineered voting methodology.

- Approval voting consistently scores highly on voter satisfaction simulations

  Approval voting is typically only outperformed by voting methodologies that are based on score/range voting (*not* ranked voting, which is different and performs worse than approval voting).  This is why approval voting is widely regarded as the voting methodology that provides the highest "bang for the buck", meaning that it produces a very high voter satisfaction while being remarkably simple.

**Comment by mschwaig** [comment]
I don't think that someone who has been banned should be excluded from voting, and if  I am elected I will try to convince the other members of the SC of my opinion on the manner, to see if we can change this.

My argument is, that I would want the SC to represent the opinions of everyone with a certain personal investment in the Nix community. I think banning people from voting feeds a certain us versus them mentality, which I think we should not feed.

Someone might be banned, because the way they talk is unacceptable.
Still, I would prefer to be able to refer those few people towards a productive process, where they can vote their opinion, maybe against my own particular interests, than turning them into the oppressed outsider.
Some people would rather be the oppressed outsider, because that's just another way to stir up drama an division.

I don't know about the practical considerations around bans and GitHub that went into this rule as is is now, though.

**Comment by tomberek** [comment]
The NCA has put in some good work creating the [constitution](https://github.com/NixOS/nix-constitutional-assembly/blob/main/constitution.md). I think we should work with it for a while and see what develops. While I might be able to come up with some improvements, I think proceeding with a generally agreed upon framework is more important than any specific change I have in mind.

**Comment by getchoo** [comment]
Similar to cafkafk above, I am interested in ensuring membership of the Foundation Board and Steering Committee is mutually exclusive. Aside from that, I can't say I have many objections or suggestions. I am very satisfied with how our first Constitution turned out, and I believe it has given the community a solid foundation for amending it in the future

**Comment by fabianhjr** [comment]
Regarding Aproval Voting @Gabriella439,

What are your thoughts on Reweighted Approval Voting / Sequential Proportional Approval Voting for multiple winners elections?

**Comment by nyabinary** [comment]
I would propose amendments to enhance community involvement and transparency. One specific change would be to introduce mechanisms for more direct community participation, such as community referendums on significant decisions and a more formalized system for feedback loops. I would also advocate for more transparent decision-making processes, ensuring that SC voting records and meeting minutes are constitutionally mandated to be publicly accessible.

**Comment by proofconstruction** [comment]
While the Constitutional Assembly was drafting the constitution, I gave some feedback which amounted to "we should separate the *legislative* power of the Steering Committee from the *executive* power of a to-be-created Executive Directorate". The Executive Directorate would be headed by the Executive Director, which would be a *paid, full-time* position, to which the leads of official teams would report. An early draft of the proposed constitutional refactor can be found [here](https://github.com/proofconstruction/constitution-changes/blob/main/executive-directorate.md).

The primary motivation behind this change is to improve accountability: there would be one central reporting structure, one person ultimately responsible for outcomes delivered by the official teams, and a final arbiter of disputes.

**Comment by Infinidoge** [comment]
I think cafkafk, Gabriella, and nyabinary all have good ideas that are worth implementing.

- Approval voting is simple, straight forward, and effective. A simpler voting method makes it easier to get more people to vote.
- Making the board and SC mutually exclusive would be a very simple change that would help prevent someone from holding too much power
- Creating avenues for the community to be directly involved in governance allows significant changes to be made without them being made from on-high, preventing a situation where the SC or similar makes a big change without the approval of the community.
- Transparency is a must, and public SC voting records and meeting minutes would need to be a part of that.

Those all covered the biggest flaws I saw while reading through it myself.

**Comment by nyabinary** [comment]
I agree with Cafkafk, Gabriella, and Infinidoge regarding the necessary constitutional changes. My main focus would be to advocate for amendments that make governance more representative of the broader community and provide direct avenues for community involvement in key decision-making processes.

I strongly support the introduction of approval voting as it simplifies the process, making participation more accessible and increasing voter turnout. Additionally, separating the board and the SC is a must in my eyes, it's a no-brainer amendment to the current constitution that will prevent the concentration of power.

Furthermore, I believe in empowering the community through referendums and structured ways for voting on important initiatives. Creating more opportunities that allow for community-led governance and input will ensure the voices of all members are heard and respected, reducing the risk of significant changes being made without community input. Transparency is another crucial element that needs **serious** improvement in my opinion. I would work towards publishing SC voting records and ensuring meeting minutes are publicly available, ensuring openness in all aspects of governance.
